### ISSUE TITLE
art-of-the-deal diacritics

### ISSUE THREAD

**Issue by CrafterSvK** [pumpkin-py/pumpkin-fun#68]
Marlboro regular does not include some letters with diacritics like `č` fallback with ASCII character would be nice, or port some letters using existing ones if that is an option.

**Comment by sinus-x** [comment]
Yeah. We need some other font that is similar to the original and is free, so we can include it in the repository.
